CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(THREADS_PREFER_PTHREAD_FLAG ON)
 
+set(CMAKE_CXX_FLAGS "-DGEOS_INLINE")
```

Comment:
you can pass that via command line to cmake

Reply:
Using the cmd line was giving me errors in my build - probably because of apple silicon's bugs, I will edit it out in the push build.